node/coinstacks/ethereum/api/src/app.ts

```diff
@@ -38,8 +45,25 @@ app.get('/', async (_, res) => {
 app.use(middleware.errorHandler)
 app.use(middleware.notFoundHandler)
 
-const server = app.listen(port, () => logger.info({ port }, 'Server started'))
+const registry = new Registry()
+  .formatAddress((address) => ethers.utils.getAddress(address))
+  .blockHandler<NewBlock, Array<BlockbookTx>>(async (block) => {
+    const txs = await handleBlock(block.hash)
+    return { txs }
+  })
+  .transactionHandler<BlockbookTx, EthereumTx>(async (blockbookTx) => {
+    const tx = await handleTransactionWithInternalTrace(blockbookTx)
```

Comment:
At first I assumed this was being used alongside the etherscan call or if the etherscan call failed but now I fee its only used here.

Maybe it would be better to rename it getInternalAddressesFromTx and put the next line in it as well?

feels more clear to me